core/src/blocktree.rs

```diff
@@ -127,6 +131,7 @@ pub struct Blocktree {
     erasure_cf: ErasureCf,
     detached_heads_cf: DetachedHeadsCf,
     pub new_blobs_signals: Vec<SyncSender<bool>>,
+    pub root_slot: RwLock<u64>,
```

Comment:
this bugger doesn't need synchronization with anything else in this struct?

Reply:
it's accessed from both repair_service and replay_stage

Reply:
only written by one thread, then?  and written after all inputs from which we derive root have been read, right?  just asking about internal consistency.

Reply:
yeah only written by replay_stage after locktower has finalized a new root.